src/components/UI/Controls/Button/index.tsx

```diff
@@ -175,7 +175,7 @@ const Button = React.forwardRef<
               buttonStyle !== ButtonStyle.Link
             }
             label={children}
-            disabled={disabled || loading}
+            disabled={disabled || loading || unauthorized}
```

Comment:
Update `Button` logic - if a button displays that a user lacks permissions (unauthorized), it should also be shown as disabled.